kuma/static/js/analytics.js

```diff
@@ -57,31 +85,36 @@
                 // append it to the data object sent to GA
                 data.nonInteraction = eventObject.nonInteraction;
             }
-
-            // If Analytics has loaded, go ahead with tracking
-            // Checking for ".create" due to Ghostery mocking of ga
-            if(ga && ga.create) {
-                // Send event to GA
-                ga('send', data);
-            }
-            else if(ga && !ga.create) {
-                // GA blocked or not yet initialized
-                // strip callback from data
-                data.hitCallback = null;
-                // add to queue without callback
-                ga('send', data);
-                // execute callback now
-                if(callback) {
-                    callback();
-                }
-            }
-            else if(callback) {
-                // GA disabled or blocked or something, make sure we still
-                // call the caller's callback:
-                callback();
-            }
+            mdn.analytics.send(data, callback);
         },
+        /**
+         * Sends a timing event to GA
+         * @param {object} timingObject - The timing data to send to GA
+         */
+        trackTiming: function(timingObject) {
+            /*
+                Format:
+                    ga('send', {
+                        'timingCategory' : 'JS Dependencies',
+                        'timingVar'      : 'load',
+                        'timingValue'    : 100,
+                        'timingLabel'    : 'interactive-examples',
+                        'hitCallback'   : function () {
+                            document.location = href;
+                        },
+                        'hitType': 'timing'
+                    });
+            */
+            var data = {
+                hitType: 'tming',
+                timingCategory: timingObject.category || '', // Required.
+                timingVar: timingObject.timingVar || '', // Required.
+                timingValue: timingObject.value || 0,  // Required.
+                timingLabel: timingObject.label || '',
```

Comment:
I think something like this would be better, to make the label optional:

```
           var data = {
               hitType: 'tming',
               timingCategory: timingObject.category || '', // Required.
               timingVar: timingObject.timingVar || '', // Required.
               timingValue: timingObject.value || 0,  // Required.
           };
           if (timingObject.label) {
               data.timingLabel = timingObject.label;
           }
```

Please forgive if my JS looks like Python.

That way, if the label is omitted or falsy it is not sent. It is possible that Google treats an empty string the same as an omitted label, but it is certain they treat an omitted key as omitted.